packages/fxa-content-server/app/scripts/views/mixins/experiment-mixin.js

```diff
@@ -57,37 +57,21 @@ const ExperimentMixin = {
       this.experiments = null;
     }
   },
-
-  /**
-   * Create an experiment and add it to the list of active experiments.
-   * Only creates an experiment with `experimentName` once.
-   * This is useful to create an experiment that is not created
-   * at startup.
-   *
-   * @param {String} experimentName - name of experiment to create.
-   * @param {String} groupType - which group the user is in.
-   * @returns {Object} experiment object, if created.
-   */
-  createExperiment(...args) {
-    if (this.experiments) {
-      // force the flow model to be initialized so that
-      // the experiment is logged.
-      this.notifier.trigger('flow.initialize');
```

Comment:
I pushed this into the ExperimentInterface which enabled me to replace this with a delegate method below.